compliance/aws/ecs_unused/README.md

```diff
@@ -12,6 +12,7 @@ When an unused ECS cluster is detected, an email action is triggered automatical
 
 ## Input Parameters
 
+- *Allowed Regions* - A list of allowed regions for an AWS account. Click [here](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions) to check regions in AWS and enter the region code. If this field is left empty, then the policy will throw an error.
```

Comment:
In chat, @cdwilhelm had mentioned:

> I don't think we should force the user to enter all the regions.  Most users will not need/or want to enter all of the regions

The behavior described here does not seem to have taken that into account. I agree with Curt that a solution where leaving the Allowed Regions list empty means that the policy uses all the regions that it finds using the `DescribeRegions` call. This part of the documentation can then make clear that if SCP is enabled and some regions are disabled there will be failures, so only the actually enabled regions should be entered into the list in that case.

Reply:
@douglaswth, I have already mentioned in the README that if this field is left empty the policy will throw an error.

Reply:
@douglaswth, I thought of adding as mentioned below. Is this fine?.

- *Allowed Regions* - A list of allowed regions for an AWS account. See [Available Regions](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html#concepts-available-regions) to check regions in AWS and enter the region code. If SCP is enabled for an AWS account then enter only the enabled regions if in case disabled regions are entered or if this field is left blank then the policy will throw an error.

Reply:
Throwing an error when the list is left empty is exactly the opposite of what Curt and I are suggesting. Forcing users who do not have SCP in place to add every single region into the list in order to get useful functionality is not a good user experience. Only requiring adding regions when SCP is enabled makes a lot more sense.

Reply:
@SheikSena  If the user leaves the Allowed Regions empty the policy should run without any error, evaluating all regions.

Reply:
@douglaswth, I made the necessary changes as suggested Can you verify once?. Thanks